NEWS.md

```diff
@@ -147,6 +147,8 @@ both UIs integration are available from https://github.com/meztez/rapidoc/ and h
 
 * Date response header is now supplied by httpuv and not plumber. Fixes non standard date response header issues when using different locales. (@shrektan, #319, #380)
 
+* An error will be thrown if multiple arguments are matched to an Plumber Endpoint route definition.  Users should define their route to only use `req`, `res`, and/or `...` to avoid multiple matchings. (#637)
```

Comment:
Add a little more context. 

* Perhaps mention you can access "special" args  (like `argsQuery`, `argsPaths`, and `argsPostBody`) and that leveraging these new args in a function signature like `function(req, res) {}`  is a much better/safer way to access and use these args (compared to putting them in the function signature).

* Maybe do the above via an example.